src/CustomElementRegistry.js

```diff
@@ -79,148 +46,60 @@ export default class CustomElementRegistry {
       throw new SyntaxError(`The element name '${localName}' is not valid.`);
     }
 
-    if (this._internals.localNameToDefinition(localName)) {
+    if (this._internals.localNameToDefinition(localName) ||
+        this._internals.localNameToLazyDefinition(localName)) {
       throw new Error(`A custom element with name '${localName}' has already been defined.`);
     }
 
-    if (this._elementDefinitionIsRunning) {
+    if (this._internals.elementDefinitionIsRunning) {
```

Comment:
It looks like `elementDefinitionIsRunning` is public only for this check; maybe this should be moved into `CustomElementInternals` so that it can remain private?

Reply:
Done